contracts/token/PCVEquityMinter.sol

```diff
@@ -0,0 +1,89 @@
+pragma solidity ^0.8.0;
+
+import "./FeiTimedMinter.sol";
+import "./IPCVEquityMinter.sol";
+import "@openzeppelin/contracts/utils/math/SafeCast.sol";
+
+/// @title PCVEquityMinter
+/// @notice A FeiTimedMinter that mints based on a percentage of PCV equity
+contract PCVEquityMinter is IPCVEquityMinter, FeiTimedMinter {
+    using SafeCast for int256;
+
+    /// @notice The maximum percentage of PCV equity to be minted per year, in basis points 
+    uint256 public constant override MAX_APR_BASIS_POINTS = 2000; // Max 20% per year
+
+    uint256 private constant SECONDS_PER_YEAR = 365 days;
+    uint256 private constant BASIS_POINTS_GRANULARITY = 10000;
+    uint256 private constant FEI_MINTING_LIMIT_PER_SECOND = 1000e18; // 1000 FEI/s or ~86m FEI/day
```

Comment:
In the RateLimitedMinter we defined the `MAX_FEI_LIMIT_PER_SECOND` as 10,000 FEI per second. Should they both be the same?

Reply:
One is a "max" the other is the actual parameter value. When changing the param it is validated that it is less than the max

Reply:
Might be good to have the minting rate limit per second changeable by governor

(mostly in case we want to lower it)

Reply:
Will make it changeable by governance in a separate PR